package.json

```diff
@@ -20,8 +20,12 @@
   "scripts": {
     "prepare": "tsc",
     "build": "tsc",
+    "test": "npm run compile && mocha dist/test",
+    "test:unit": "nyc --reporter=text-summary --reporter=cobertura --report-dir=reports  mocha --reporter mocha-junit-reporter --reporter-options mochaFile=./reports/unit/junit-test-results.xml dist/test",
+    "compile": "tsc -b",
```

Comment:
can you explain the build flag?

Reply:
rather than my bad explanation : https://www.typescriptlang.org/docs/handbook/project-references.html

TLDR: faster ( incremental builds)

Reply:
we already get incremental builds by tsconfig

Reply:
which isn't present on all projects sadly

Reply:
it is not limited to incremental builds.